### ISSUE TITLE
"Error: unknown database engine detected from file" when importing from phpMyAdmin

### ISSUE THREAD

**Issue by jamiematrix** [craftcms/nitro#269]
### Description
I'm desperately trying to replace MAMP and Nitro 2 is looking just the thing! I was hoping to migrate existing sites over to Nitro based ones so thought I'd do the usual phpMyAdmin export of an existing DB then import into a Nitro site.
No joy. Have tried a couple of different DBs (even a fresh Craft install) and get the above error.

I'm using the phpMyAdmin export as WHEN I get Nitro running on my Windows machine, I'm not able to export Craft DB's via Craft admin (I think it's MAMP that cannot find the SQLDUMP application).

### Steps to reproduce

1. Export DB using phpMyAdmin
2. Use Nitro CLI to import the SQL file

### Additional info

- Nitro version: 2.0.3
- Docker version: 3.1.0

**Comment by jasonmccallister** [comment]
@jamiematrix can you provide the file extension and the first 50 lines of the database file? We try to detect the engine (mysql or postgres) before running the import.

The first 50 lines should include some keywords we are looking for.

**Comment by jamiematrix** [comment]
Extension is .sql
First 50 lines:
```
-- phpMyAdmin SQL Dump
-- version 4.9.5
-- https://www.phpmyadmin.net/
--
-- Host: localhost:3306
-- Generation Time: Mar 02, 2021 at 05:48 PM
-- Server version: 5.7.24
-- PHP Version: 7.3.7

SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
SET AUTOCOMMIT = 0;
START TRANSACTION;
SET time_zone = "+00:00";


/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8mb4 */;

--
-- Database: `nitrotest_craft`
--

-- --------------------------------------------------------

--
-- Table structure for table `assetindexdata`
--

CREATE TABLE `assetindexdata` (
  `id` int(11) NOT NULL,
  `sessionId` varchar(36) NOT NULL DEFAULT '',
  `volumeId` int(11) NOT NULL,
  `uri` text,
  `size` bigint(20) UNSIGNED DEFAULT NULL,
  `timestamp` datetime DEFAULT NULL,
  `recordId` int(11) DEFAULT NULL,
  `inProgress` tinyint(1) DEFAULT '0',
  `completed` tinyint(1) DEFAULT '0',
  `dateCreated` datetime NOT NULL,
  `dateUpdated` datetime NOT NULL,
  `uid` char(36) NOT NULL DEFAULT '0'
) ENGINE=InnoDB DEFAULT CHARSET=utf8;

--
-- Dumping data for table `assetindexdata`
--

INSERT INTO `assetindexdata` (`id`, `sessionId`, `volumeId`, `uri`, `size`, `timestamp`, `recordId`, `inProgress`, `completed`, `dateCreated`, `dateUpdated`, `uid`) VALUES
```
I've found a work around by creating the DB using TablePlus, then importing there. Most of the time it seems to work and connect (a few times Craft would't connect to MySQL)

**Comment by jasonmccallister** [comment]
Ok, thank you. I will work on getting a fix out for this ASAP (most likely in the next couple of days).

**Comment by jasonmccallister** [comment]
This was released with 2.0.4 so it should detect your database engine now, it will also not throw an error if it cannot determine the engine.
